MaterialSkin/Plugin.pm

```diff
@@ -186,17 +191,17 @@ sub _cliCommand {
         my $osDetails = Slim::Utils::OSDetect::details();
         my $serverPrefs = preferences('server');
         $request->addResult('info', '{"server":'
-                                .'[ {"label":"' . cstring('', 'INFORMATION_VERSION') . '", "text":"' . $::VERSION . ' - ' . $::REVISION . ' @ ' . $::BUILDDATE . '"},'
-                                .  '{"label":"' . cstring('', 'INFORMATION_HOSTNAME') . '", "text":"' . Slim::Utils::Network::hostName() . '"},'
-                                .  '{"label":"' . cstring('', 'INFORMATION_SERVER_IP') . '", "text":"' . Slim::Utils::Network::serverAddr() . '"},'
-                                .  '{"label":"' . cstring('', 'INFORMATION_OPERATINGSYSTEM') . '", "text":"' . $osDetails->{'osName'} . ' - ' . $serverPrefs->get('language') .
+                                .'[ {"label":"' . cstring($client, 'INFORMATION_VERSION') . '", "text":"' . $::VERSION . ' - ' . $::REVISION . ' @ ' . $::BUILDDATE . '"},'
```

Comment:
What if there is no client? For this 'info' command no client is specified.

Reply:
That’s what cstring() is for: it checks, whether a client is defined or not. If you know there’s none, then just use string() instead.